### ISSUE TITLE
Problem with directories and containers volumes with the docker-compose file provided.

### ISSUE THREAD

**Issue by matissime** [aleksasiriski/rffmpeg-go#6]
Hello,

I just tried to switch my existing Jellyfin instance to your own image by using the [docker-compose file you provided](https://github.com/aleksasiriski/rffmpeg-go/blob/main/docker-compose.example.yml) and I set my own working directories for Jellyfin but I keep getting this error with the nfs share container : 

nfs-share         | ==================================================================
nfs-share         |       SETTING UP ...
nfs-share         | ==================================================================
nfs-share         | ----> building /etc/exports from environment variables
nfs-share         | ----> WARNING: skipping NFS_EXPORT_0 environment variable since "/var/jellyfin/cache is not a container directory
nfs-share         | ----> WARNING: skipping NFS_EXPORT_1 environment variable since "/var/jellyfin/transcodes is not a container directory
nfs-share         | ----> WARNING: skipping NFS_EXPORT_2 environment variable since "/var/media/subtitles is not a container directory
nfs-share         | ----> WARNING: skipping NFS_EXPORT_3 environment variable since "/var/media/movies is not a container directory
nfs-share         | ----> WARNING: skipping NFS_EXPORT_4 environment variable since "/var/media/shows is not a container directory
nfs-share         | ----> collected 0 valid export(s) from NFS_EXPORT_* environment variables
nfs-share         | ----> 
nfs-share         | ----> ERROR: no valid exports
nfs-share         | ---->

And here's my docker-compose file :

version: '3.9'
services:
  jellyfin-rffmpeg:
    image: ghcr.io/aleksasiriski/jellyfin-rffmpeg:latest
    environment:
      - TZ=Europe/Paris
    container_name: jellyfin-rffmpeg
    user: 1001:1001
    network_mode: 'host'
    ports:
      - 8096:8096
    volumes:
      - /var/jellyfin/config:/config:Z
      - /var/jellyfin/cache:/config/cache:z
      - /var/jellyfin/transcodes:/config/transcodes:z
      - /var/media/subtitles:/config/data/subtitles:z
      - /var/media/movies:/movies:ro
      - /var/media/shows:/shows:ro
    restart: always
  nfs-share:
    image: ghcr.io/obeone/nfs-server:latest
    container_name: nfs-share
    user: 1001:1001
    network_mode: 'host'
    cap_add:
      - SYS_ADMIN
    environment:
      - TZ=Europe/Paris
      - NFS_EXPORT_0="/var/jellyfin/cache           *(rw,sync,no_subtree_check,no_root_squash)"
      - NFS_EXPORT_1="/var/jellyfin/transcodes      *(rw,sync,no_subtree_check,no_root_squash)"
      - NFS_EXPORT_2="/var/media/subtitles  *(rw,sync,no_subtree_check,no_root_squash)"
      - NFS_EXPORT_3="/var/media/movies      *(rw,sync,no_subtree_check,no_root_squash)"
      - NFS_EXPORT_4="/var/media/shows      *(rw,sync,no_subtree_check,no_root_squash)"
    ports:
      - 2049:2049
    volumes:
      - jellyfin-cache:/var/jellyfin/cache:z
      - jellyfin-transcodes:/var/jellyfin/transcodes:z
      - jellyfin-subtitles:/var/media/subtitles:z
      - /var/media/movies:/var/media/movies:ro
      - /var/media/shows:/var/media/shows:ro
    restart: always
volumes:
  jellyfin-conf:
  jellyfin-cache:
  jellyfin-transcodes:
  jellyfin-subtitles:

So my working directories are :
For jellyfin : /var/jellyfin/{config,cache,transcodes}
And for the media files (it's a mount point) : /var/media/{movies,shows,subtitles}
And those directories should be mapped to the directories that jellyfin use. But it don't work as expected.

I tried multiples things such as removing the volumes sections at the end and re-linked the container variables to the directories.

**Comment by aleksasiriski** [comment]
I'll give this a look when I get some free time.

**Comment by matissime** [comment]
Okay, It was due to the quotations in the environnements variables of the nfs server. Ref: [ehough/docker-nfs-server: Method 2 to provide exports fails](https://github.com/ehough/docker-nfs-server/issues/13#issuecomment-437125456)

So it's this line that need to be changed from this:

```
environment:
- TZ=Europe/Paris
- NFS_EXPORT_0="/var/jellyfin/cache *(rw,sync,no_subtree_check,no_root_squash)"
- NFS_EXPORT_1="/var/jellyfin/transcodes *(rw,sync,no_subtree_check,no_root_squash)"
- NFS_EXPORT_2="/var/media/subtitles *(rw,sync,no_subtree_check,no_root_squash)"
- NFS_EXPORT_3="/var/media/movies *(rw,sync,no_subtree_check,no_root_squash)"
- NFS_EXPORT_4="/var/media/shows *(rw,sync,no_subtree_check,no_root_squash)"
```

To this (just remove the "quotes" markers):

```
environment:
- TZ=Europe/Paris
- NFS_EXPORT_0=/var/jellyfin/cache *(rw,sync,no_subtree_check,no_root_squash)
- NFS_EXPORT_1=/var/jellyfin/transcodes *(rw,sync,no_subtree_check,no_root_squash)
- NFS_EXPORT_2=/var/media/subtitles *(rw,sync,no_subtree_check,no_root_squash)
- NFS_EXPORT_3=/var/media/movies *(rw,sync,no_subtree_check,no_root_squash)
- NFS_EXPORT_4=/var/media/shows *(rw,sync,no_subtree_check,no_root_squash)
```

**Comment by aleksasiriski** [comment]
Fixed by #7
